test/Microsoft.AspNet.Localization.FunctionalTests/TestRunner.cs

```diff
@@ -60,10 +65,52 @@ public class TestRunner
                 }
             }
         }
-
-        public string GetApplicationPath()
+        
+        public async Task RunTestAndVerifyResponseHeading(
```

Comment:
You 're right, at that time I don't wanna break anything in the previous tests, but I will try to share the code between them